File: sdkwork-backend-sdk-go/types/plus_sharding_key_form.go
package types

// Sharding key creation form
type PlusShardingKeyForm struct {
	ShardingKey string `json:"shardingKey"`
	TenantId int `json:"tenantId"`
	Owner string `json:"owner"`
	OwnerId int `json:"ownerId"`
	Status string `json:"status"`
	Description string `json:"description"`
}

File: sdkwork-backend-sdk-typescript/src/types/plus-sharding-key-form.ts
/** Sharding key creation form */
export interface PlusShardingKeyForm {
  /** Sharding key */
  shardingKey: string;
  /** Tenant ID */
  tenantId: number;
  /** Owner type */
  owner: 'DEFAULT' | 'TENANT' | 'ORGANIZATION';
  /** Owner ID */
  ownerId: number;
  /** Status */
  status: 'DEFAULT' | 'ACTIVE' | 'INACTIVE' | 'SUSPENDED' | 'DELETED';
  /** Description */
  description?: string;
}
